tlsn/examples/discord/discord_dm.rs

```diff
@@ -197,8 +195,12 @@ async fn main() {
 
 async fn setup_notary_connection() -> (tokio_rustls::client::TlsStream<TcpStream>, String) {
     // Connect to the Notary via TLS-TCP
-    let mut certificate_file_reader = read_pem_file(NOTARY_CA_CERT_PATH).await.unwrap();
-    let mut certificates: Vec<Certificate> = rustls_pemfile::certs(&mut certificate_file_reader)
+    let pem_file = str::from_utf8(include_bytes!(
+        "../../../notary-server/fixture/tls/rootCA.crt"
```

Comment:
might be easier to remain this as a global constant at the top for easier configuration if one decides to change the cert?